pkg/client/generated/project/v3/zz_generated_claim_source.go

```diff
@@ -1,12 +0,0 @@
-package client
-
-const (
-	ClaimSourceType                           = "claimSource"
-	ClaimSourceFieldResourceClaimName         = "resourceClaimName"
-	ClaimSourceFieldResourceClaimTemplateName = "resourceClaimTemplateName"
-)
-
-type ClaimSource struct {
```

Comment:
We're removing this struct, which was introduced upstream in 1.26 and removed in upstream 1.31. It's part of dynamic resource allocation, which remains an alpha feature, so this should be fine.

Dynamic resource allocation is an alpha feature and only functions when the DynamicResourceAllocation [feature gate](https://kubernetes.io/docs/reference/command-line-tools-reference/feature-gates/) and the resource.k8s.io/v1alpha3 [API group](https://kubernetes.io/docs/concepts/overview/kubernetes-api/#api-groups-and-versioning) are enabled.